dotnetv3/S3/README.md

```diff
@@ -0,0 +1,92 @@
+# Amazon Simple Storage Service (Amazon S3) code examples in C#
```

Comment:
C# -> .Net (right?)

In title, use 'Amazon S3'

Reply:
You're right. Fixed.